bg/lessons/basics/control-structures.md

```diff
@@ -91,7 +91,7 @@ iex> case {1, 2, 3} do
 "Will match"
 ```
 
-Проветеве официалната документация за [Допустими изрази в предпазващи клаузи](http://elixir-lang.org/getting-started/case-cond-and-if.html#expressions-in-guard-clauses).
+Проветеве официалната документация за [Допустими изрази в предпазващи клаузи](https://hexdocs.pm/elixir/master/guards.html).
```

Comment:
Thanks, @sgnl!
I am not sure whether we should link to the `master` docs directly. Isn't [`stable`](https://hexdocs.pm/elixir/guards.html) a better choice?

Reply:
I think that's a good idea @nscyclone 👍